icu4j/main/classes/core/src/com/ibm/icu/number/Precision.java

```diff
@@ -552,13 +552,18 @@ int chooseMultiplierAndApply(DecimalQuantity input, MultiplierProducer producer)
     ///////////////
 
     /**
-     * An BogusRounder's MathContext into precision.
+     * BogusRounder does not support rounding, but holds a MathContext which can
+     * be copied into a proper Precision object at a later time.
      *
      * @internal
      * @deprecated This API is ICU internal only.
      */
     @Deprecated
     public static class BogusRounder extends Precision {
```

Comment:
Still need to figure out what this one is missing:

  [javadoc] warning: no tag for com.ibm.icu.number.Precision.BogusRounder()